data/remnant/remnant jobs.txt

```diff
@@ -659,3 +659,36 @@ mission "Remnant: Rescue 5"
 	on visit
 		dialog
 			`You have returned, but there is still no word from the missing ship.`
+
+
+mission "Remnant: Disable 1"
+	job
+	repeat
+	name "Disable Damaged Kas'lor Ik 582"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Will Not Someone Please Think Of The Children: done"
```

Comment:
It says below if you abort or fail that it will not be offered again, but it'll offer again anyway.

```suggestion
	to offer
		has "Remnant: Will Not Someone Please Think Of The Children: done"
		not "Remnant: Disable 1: aborted"
		not "Remnant: Disable 1: failed"
```

Reply:
I'm still dubious on if we should offer the WS on a plate (its defenceless)and then not offer it again, caus most ppl would just cap it.
My idea when proposing the mission was to encourage looting but I wouldnt mind capping it I guess (also it being in postverta means its locked until we unlock it and that exiles know of it)

Reply:
For context, ppl were willing to halt the entire remnant story to get a garanteed WS. Granted this one is cargo and not crew but still.

Reply:
The 582 can now be captured and extracted from Korath space. So I think this is an ok trade off.